src/EDA.py

```diff
@@ -0,0 +1,81 @@
+"""This script saves EDA charts.
+Usage: EDA.py <path_to_x_train> <path_to_y_train> <loc_to_be_saved>
+
+Options:
+<path_to_x_train>             path to X_train.csv
+<path_to_y_train>             path to y_train.csv
+<loc_to_be_saved>             path to where we want to save the images
+
+"""
```

Comment:
Can you create an example here like:

Example:
python ml_models.py data/train.csv, results/raw_results